libcxx/test/std/diagnostics/syserr/syserr.syserr/syserr.syserr.members/ctor_error_code_string.pass.cpp

```diff
@@ -21,11 +21,12 @@
 #include "test_macros.h"
 
 int main(int, char**) {
-    std::string what_arg("test message");
+    std::string what_arg("test message\0With embedded NUL");
```

Comment:
Can you instead add a new test that explicitly mentions the LWG issue? I'm worried that this coverage could go away via a refactoring otherwise.